screens/ChatsScreen.js

```diff
@@ -28,14 +28,38 @@ import {
     Item,
     Input
 } from 'native-base';
-import { fetchChats, setCurrentChat } from '../actions/chats';
+import { fetchChats, setCurrentChat, putUserInChat } from '../actions/chats';
+import { setCurrentUser } from '../actions/user';
 
 class ChatsScreen extends React.Component {
     static navigationOptions = {
         header: null
     };
 
+    // Get user info from AsyncStorage and store in redux state
+    getUser = async () => {
+        try {
+            const storedUser = await AsyncStorage.getItem('user');
+            const user = JSON.parse(storedUser); 
+            if (user) {
+                const { id, display_name } = user;
+                return {
+                    id,
+                    display_name
+                };
+            }
+        } catch (error) {
+            console.log(error);
+            return Promise.reject({ message: 'Error getting data' });
+        }
+
+    }
+
     componentDidMount() {
+        // If current user object is empty, populate
+        if (Object.getOwnPropertyNames(this.props.currentUser).length === 0) {
```

Comment:
i believe you can just do:

if (!this.props.currentUser)

Reply:
Ah I see, I was thinking I was doing something unnecessary here--I set the currentUser's initial state to {} in which you would have to use some kind of check like the above since it's just an empty object and not null, but I can easily just change it to null initially and then use !this.props.currentUser.